packages/compass-components/src/components/document-list/document-actions-group.tsx

```diff
@@ -44,6 +44,13 @@ const actionsGroupSignalPopover = css({
   display: 'block !important',
 });
 
+const expandButton = css({
+  '& > div:has(svg)': {
```

Comment:
This does feel brittle, if something changes in how leafygreen orientates their buttons internally this could start looking different.  Not sure if there's a better way to do this though, looking at the docs I don't think this is  exposed for folks. Looks fine, mostly calling it out in case there's a way anyone else knows of, and also something we can watch out for when updating leafygreen deps.

Reply:
Yeah - I agree this is brittle. I should have made that more obvious in my PR: I've proposed a change to LG to upstream this https://github.com/mongodb/leafygreen-ui/pull/2535.